examples/howto/plot_record_extracellular_potentials.py

```diff
@@ -23,18 +23,31 @@
 import matplotlib.pyplot as plt
 
 ###############################################################################
-# We will use the default network with three evoked drives; see
-# :ref:`evoked example <sphx_glr_auto_examples_plot_simulate_evoked.py>` for
-# details. We'll go ahead and use the drive features defined in the parameter
-# file.
+# This example relies on the default "Jones2009"-network.
 
 import hnn_core
 from hnn_core import read_params, jones_2009_model, simulate_dipole
+from hnn_core.network_models import add_erp_drives_to_jones_model
 
 hnn_core_root = op.dirname(hnn_core.__file__)
 params_fname = op.join(hnn_core_root, 'param', 'default.json')
 params = read_params(params_fname)
-net = jones_2009_model(params, add_drives_from_params=True)
+
+###############################################################################
+# The default network model defined in Jones et al. (2009) [1]_ consists of a
+# square grid of pyramidal cells. The in-plane distance between pyramidal cell
+# somas on the grid can be set by the user, which will have an influence on the
+# extracellular potentials (but not on the calculated net intracellular dipole
+# moment). In this example, we'll simulate a network of model cells spaced
+# 30 um apart. To drive the network dynamics, we'll use three evoked 'ERP'
+# drives; see
+# :ref:`evoked example <sphx_glr_auto_examples_plot_simulate_evoked.py>`
```

Comment:
I think this will not render as a link, you might have to change this because the folder hierarchy changed and this is now under tutorials